crd/src/lib.rs

```diff
@@ -27,11 +29,35 @@ impl Crd for KafkaCluster {
     const CRD_DEFINITION: &'static str = include_str!("../kafkaclusters.crd.yaml");
 }
 
-#[allow(non_camel_case_types)]
 #[derive(Clone, Debug, Deserialize, JsonSchema, Serialize)]
-pub enum KafkaVersion {
-    #[serde(rename = "2.6.0")]
-    v2_6_0,
+pub struct KafkaVersion {
+    kafka_version: String,
+    scala_version: Option<String>,
+}
+
+impl KafkaVersion {
+    pub const DEFAULT_SCALA_VERSION: &'static str = "2.13";
```

Comment:
The scala version is irrelevant for most users of Kafka, which is why I do not really want to make it mandatory. The Kafka project publishes a recommendation of which version to pick if you don't care - which currently is 2.13 for all versions of Kafka supported by Stackable, so I took the easy way out here :)
I can implement a lookup based on the version to be prepared for a future change in the recommendation of course.

Reply:
This decision is up to you.

Reply:
I'll leave it as is for now so that we can make progress. I have implemented a lookup of the recommended version, but would like to discuss the best way forward when there is more time.
I'll open a draft PR right after this, so it is not forgotten.